### PR TITLE
DevWs for NIE-TECH CO., LTD. containing containing Z-Wave Metering Switch

### PR DESCRIPTION
**Pull request by sky-nie**
In the DTH file "Z-Wave Metering Switch", we have added support for a new product (MP22ZP).

### CODE REVIEW

**Comment by sky-nie** [comment]
In the DTH file "Z-Wave Metering Switch", we have added support for a new product (MP22ZP).

File: devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy [hunk 1]
```diff
@@ -284,10 +285,6 @@ def configure() {
 }
 
 def reset() {
-	resetEnergyMeter()
-}
-
-def resetEnergyMeter() {
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy]: Why did You change that ?

**Review by KKlimczukS — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy (hunk 1 above)
Comment (sky-nie) [reply devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy]: Sorry, I made the changes based on the master, but I forgot that this master was forked by me, not the latest.

File: devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy [hunk 2]
```diff
@@ -53,6 +53,8 @@ metadata {
 		fingerprint mfr: "0154", prod: "0003", model: "000A", deviceJoinName: "POPP Outlet", ocfDeviceType: "oic.d.smartplug" //EU //POPP Smart Outdoor Plug
 		fingerprint mfr: "010F", prod: "1F01", model: "1000", deviceJoinName: "Fibaro Outlet", ocfDeviceType: "oic.d.smartplug" //EU //Fibaro walli Outlet //Fibaro Outlet
 		fingerprint mfr: "0312", prod: "FF00", model: "FF0E", deviceJoinName: "Minoston Outlet", ocfDeviceType: "oic.d.smartplug" //Mini Smart Plug Meter, MP21ZP
+		fingerprint mfr: "0312", prod: "FF00", model: "FF0F", deviceJoinName: "Minoston Outlet", ocfDeviceType: "oic.d.smartplug" //Mini Smart Plug Meter, MP22ZP
+		fingerprint mfr: "0312", prod: "FF00", model: "FF11", deviceJoinName: "Minoston Outlet", ocfDeviceType: "oic.d.smartplug" //Mini Power Meter Plug, ZW38M
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy]: The developer workspace (DevWs) request says about models: MP21ZP, MP22ZP, but You added here also another model ZW38M. Was it intentional ? Are You going to open another request in DevWS for ZW38M ?

Comment (sky-nie) [reply devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy]: Because MP22ZP and ZW38M are under the brand "Minoston", I will  open another request in DevWS for ZW38M without DTH changed.

**Review by KKlimczukS — APPROVED** [review]
